framework-plugins/lisk-framework-http-api-plugin/swagger.yml

```diff
@@ -66,3 +47,768 @@ info:
+        type: string
+        example: 'k9AP5b5w2Q564keTai59g7UICcebc/oUKF8CyEI0iz4='
+        description: Base64 encoded value of the network identifier as per LIP-0009.
+      state:
+        type: string
+        example: 'connected'
+        enum:
+          - connected
+          - disconnected
+        description: |
+          The state of the peer.
+      options:
+        type: object
+        description: Optional field defined by application
+
+  NodeInfo:
+    type: object
+    required:
+      - version
+      - protocolVersion
+      - networkID
+      - lastBlockID
+      - height
+      - chainMaxHeightFinalized
+      - syncing
+      - unconfirmedTransactions
+      - genesisConfig
+    properties:
+      version:
+        type: string
+        format: version
+        example: v0.8.0
+        description: The application version that the node is running on.
+      protocolVersion:
+        type: string
+        example: 1.1
+        description: The protocol version of Lisk Core that the peer node runs on.
+      networkID:
+        type: string
+        example: 'k9AP5b5w2Q564keTai59g7UICcebc/oUKF8CyEI0iz4='
+        description: Base64 encoded value of the network identifier as per LIP-0009.
+      lastBlockID:
+        description: ID of the last processed block.
+        type: string
+        example: vPFBr0ZTsyP9hUwKaQD8dW2lKBpgMWkRdgrTUUmBBCE=
+      height:
+        type: integer
+        example: 123
+        minimum: 1
+        description: |
+          Current block height of the node.
+          Represents the current number of blocks in the chain on the node.
+      chainMaxHeightFinalized:
+        type: integer
+        example: 123
+        description: |
+          The largest height with precommits by at least 68 delegates.
+          See https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md
+      syncing:
+        type: boolean
+        example: false
+        description: True if the node is syncing with other peers.
+      unconfirmedTransactions:
+        type: integer
+        example: 1
+        minimum: 0
+        description: |
+          Number of unprocessed transactions in the pool.
+      genesisConfig:
+        type: object
+        required:
+          - blockTime
+          - maxPayloadLength
+          - rewards
+          - activeDelegates
+          - standbyDelegates
+          - delegateListRoundOffset
+        properties:
+          blockTime:
+            type: integer
+            minimum: 2
+            description: Block time interval in seconds.
+          maxPayloadLength:
+            type: integer
+            description: Maximum size of transactions allowed per block.
+          activeDelegates:
+            type: integer
+            description: Number of active delegates per round.
+          standbyDelegates:
+            type: integer
+            description: Number of standby delegates per round.
+          delegateListRoundOffset:
+            type: integer
```

Comment:
Please set `example` for each of these attributes with the default values we have.